app/models/resource.rb

```diff
@@ -8,4 +12,39 @@ class Resource < ActiveRecord::Base
   has_many :services
   has_many :ratings
   has_many :change_requests
+
+  algoliasearch per_environment: true do
```

Comment:
Does this have to be in the model? In general I like to keep model objects as spare as possible, as it keeps the application more orderly and often makes things easier to test, but in this case I see why it might be easiest. Could you perhaps have a separate search object that wraps the Resource, maybe?